components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -197,6 +198,13 @@ class Session(
         }
     }
 
+    /**
+     * The elapsed time of the load.
+     */
+    var elapsedTimeMS: Int by Delegates.observable(0) {
+        _, old, new -> notifyObservers(old, new) { onElapsedTimeMS(this@Session, new) }
+    }
```

Comment:
This part I do not understand without context: Why do we want to expose this on `Session`? :)

Also: Same naming nit as for the callback method.

Reply:
As far as I understand it, this is _the_ path from the underlying browser engine to the upper-level components. That is why I am exposing this on `Session`. The `Session` that receives these notifications can correlate that with the page that was loaded and associate the incoming data with the appropriate page load for generating the proper metrics. At least that was the goal. I am assuming that, given your comment, there is another path for surfacing this data that you would prefer that I use? I am definitely deferring to your expertise and was just trying to do _the right thing_ as far as I could grok the code!